bigframes/dtypes.py

```diff
@@ -42,6 +44,12 @@
     pd.ArrowDtype,
     gpd.array.GeometryDtype,
 ]
+ExpressionType = Union[Dtype, None]
```

Comment:
[nit] typing.Optional[Dtype] might be more concise

Reply:
Done